--- lxqt-policykit-0.8.0/CMakeLists.txt.unify	2014-12-20 12:36:10.940415893 -0200
+++ lxqt-policykit-0.8.0/CMakeLists.txt	2014-12-20 12:36:35.700512886 -0200
@@ -41,7 +41,7 @@
     if(USE_QT5)
         find_package(Qt5Widgets REQUIRED QUIET)
         find_package(Qt5LinguistTools REQUIRED QUIET)
-        find_package(lxqt-qt5 REQUIRED QUIET)
+        find_package(lxqt REQUIRED QUIET)
         find_package(qt5xdg REQUIRED QUIET)
         set(QTX_INCLUDE_DIRS ${QT_QTCORE_INCLUDE_DIR} ${QT_QTGUI_INCLUDE_DIR})
         set(QTX_LIBRARIES ${QT_QTCORE_LIBRARY} ${QT_QTGUI_LIBRARY})
